### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Added wishlist page

image

### CODE REVIEW

**Review by AurobindoGupta — APPROVED** [review]
nice work

**Comment by vivek-guda** [comment]
Good to merge. GOOD JOB!

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                <div class="card__price-wrapper">
+                  <span class="card__price-original">₹799</span>
+                  <span class="card__price-discounted">₹599</span>
+                  <span class="card__discount highlight">25% OFF</span>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="card__footer">
+            <button class="btn btn--outline-secondary">
```
Comment (saifudinbadani) [inline pages/wishlist.html:159]: please use same method everywhere for naming css classes , otherwise looks good to merge.

**Review by saifudinbadani — APPROVED** [review]

**Review by SHIKHASINGH1506 — APPROVED** [review]
All good

**Pull request merged**